packages/ui-kit/src/components/PieChart/PieChart.tsx

```diff
@@ -96,26 +98,40 @@ export const PieChartComponent = React.forwardRef<HTMLDivElement, PieChartProps>
       [theme]
     )
 
+    const totalValue = isStatic ? rows?.reduce((a, b) => a + Number(b[1]), 0) ?? 0 : Number(counterData?.counter?.value)
+
     const renderChart = React.useCallback(
       (data?: PieChartData) => {
         if (!canvasRef.current || !data || !theme || !chartConfig) {
           return
         }
 
-        const { chartColorPalette = defaultChartColorPalette, legendPosition = 'top', hideLegend = false } = chartProps
+        const {
+          chartColorPalette = defaultChartColorPalette,
+          legendPosition = 'top',
+          hideLegend = false,
+          hideTotal = false,
+          totalPosition = 'bottom'
+        } = chartProps
 
         const labels = data.rows?.map((row) => row[0]) ?? []
 
         const values = data.rows?.map((row) => Number(row[1])) ?? []
 
         const customChartLabelsPlugin: Plugin<'pie' | 'doughnut'> = getCustomChartLabelsPlugin({
-          theme
+          theme,
+          hideTotal
         })
 
         const customPlugins = {
           customCanvasBackgroundColor: {
             color: card ? theme?.bgPrimary : 'transparent'
           },
+          title: {
+            display: isPie && !hideTotal,
+            text: `Total: ${totalValue.toLocaleString()}`,
+            position: totalPosition
+          },
```

Comment:
This will work good for now, maybe we can think about enabling a prop to customize this